src/uvw/fs_event.hpp

```diff
@@ -134,8 +134,8 @@ class FsEventHandle final: public Handle<FsEventHandle, uv_fs_event_t> {
      * @param path The file or directory to be monitored.
      * @param watch Additional flag to control the behavior.
      */
-    void start(std::string path, Watch watch) {
-        start(std::move(path), Flags<Watch>{watch});
+    void start(std::string path, Event watch) {
```

Comment:
You are very welcome, uvw is a nice little wrapper. I'd be happy to add my project as soon as I publish the code (so far, it is still in early experimental stages). Keep up the great work!